src/presets/next.ts

```diff
@@ -48,6 +48,9 @@ export default {
             hover: "#f78d1b",
             active: "#f78d1b",
           },
+          mainSubtle: {
```

Comment:
nit: I think we should nest this so we keep all the tokens as lowercase

Reply:
[d056374](https://github.com/isomerpages/isomer-components/pull/49/commits/d05637412fedf3fc85b412ffadfe98b28538e97d)